StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1620GenericTypeParameterDocumentationMustMatchTypeParameters.cs

```diff
@@ -28,13 +34,16 @@ public class SA1620GenericTypeParameterDocumentationMustMatchTypeParameters : Di
         /// </summary>
         public const string DiagnosticId = "SA1620";
         private const string Title = "Generic type parameter documentation must match type parameters";
-        private const string MessageFormat = "TODO: Message format";
+        private const string MessageFormat = "{0}";
```

Comment:
You can create two `DiagnosticDescriptor` instances that have the same ID but different message formats. Only return one of them from `SupportedDiagnostics` but the other can still be used when the second message is needed.